tf2_eigen/include/tf2_eigen/tf2_eigen.h

```diff
@@ -29,46 +29,55 @@
 #ifndef TF2_EIGEN_H
 #define TF2_EIGEN_H
 
+#include <chrono>
+
 #include <tf2/convert.h>
 #include <Eigen/Geometry>
-#include <geometry_msgs/PointStamped.h>
-#include <geometry_msgs/PoseStamped.h>
+#include <geometry_msgs/msg/point_stamped.hpp>
+#include <geometry_msgs/msg/pose_stamped.hpp>
 
 
 namespace tf2
 {
-    
-Eigen::Affine3d transformToEigen(const geometry_msgs::TransformStamped& t) {
+
+Eigen::Affine3d transformToEigen(const geometry_msgs::msg::TransformStamped& t) {
   return Eigen::Affine3d(Eigen::Translation3d(t.transform.translation.x, t.transform.translation.y, t.transform.translation.z)
-			 * Eigen::Quaterniond(t.transform.rotation.w, 
+			 * Eigen::Quaterniond(t.transform.rotation.w,
 					      t.transform.rotation.x, t.transform.rotation.y, t.transform.rotation.z));
 }
 
 
 // this method needs to be implemented by client library developers
 template <>
-void doTransform(const tf2::Stamped<Eigen::Vector3d>& t_in, 
+void doTransform(const tf2::Stamped<Eigen::Vector3d>& t_in,
 		 tf2::Stamped<Eigen::Vector3d>& t_out,
-		 const geometry_msgs::TransformStamped& transform) {
+		 const geometry_msgs::msg::TransformStamped& transform) {
+  std::chrono::time_point<std::chrono::system_clock, std::chrono::nanoseconds> converted{std::chrono::nanoseconds{transform.header.stamp.sec*1000000000LL+transform.header.stamp.nanosec}};
+  std::chrono::system_clock::time_point recovered = std::chrono::time_point_cast<std::chrono::system_clock::duration>(converted);
   t_out = tf2::Stamped<Eigen::Vector3d>(transformToEigen(transform) * t_in,
```

Comment:
Sorry for being unclear I meant wrapping.
In this particular case I was referring to wrapping after opening parenthesis. This way if you decide to change `t_out` to `t_out_new` you dont need to reindent the all block.
```c++
t_out = tf2::Stamped<Eigen::Vector3d>(
  transformToEigen(transform) * t_in,
  recovered,
  transform.header.frame_id);
```
My point being that if you decide to do style changes in this PR, I'd prefer them to span the entire file / package and matching the style described in the [developer guide](https://github.com/ros2/ros2/wiki/Developer-Guide).